ringpop.go

```diff
@@ -89,8 +89,7 @@ type Ringpop struct {
 		sync.RWMutex
 	}
 
-	logger log.Logger
-	log    log.Logger
+	moduleLogger *util.ModuleLogger
```

Comment:
I think it hints that it's more than a simple logger; but I don't like ModuleLogger either.